src/components/icon.js

```diff
@@ -0,0 +1,40 @@
+(() => ({
+  name: 'Icon',
+  icon: 'ContainerIcon',
+  category: 'LAYOUT',
+  type: 'CONTENT_COMPONENT',
+  allowedTypes: [],
+  orientation: 'HORIZONTAL',
+  jsx: (() => {
+    const { Icons } = window.MaterialUI;
+    const isDev = B.env === 'dev';
+    const { icon } = options;
+    const IconComponent = React.createElement(Icons[icon], {
+      className: classes.root,
+    });
+
+    return isDev ? <div>{IconComponent}</div> : IconComponent;
```

Comment:
This `<div>` which is needed on the canvas for the drag and drop, has display `block` as default, which means that you can't have to icons next to eachother. In runtime however, this will be just an svg which has display `inline` by default, but an override to `inline-block` by material. So I suggest adding a class to the wrapping DIV on the canvas which has also display `inline-block`.